### PR TITLE
Remove dist only use public dir

### PR DESCRIPTION
**Pull request by easternbloc**
To reduce confusion we now just create a public dir and mount to that in dev mode.

Updated the package.json to do all of this post install.

Updated the run.sh to reflect these changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,17 +6,20 @@
...
     "test:acceptance": "cd acceptance_tests/; bundle install && cucumber",
     "test:ci": "npm run lint && npm run style && npm run test",
     "lint": "eslint .",
-    "copy:images": "mkdir -p ./dist; cp -r ./assets/images ./dist/images",
+    "copy:images": "cp -r ./assets/images ./public/images",
     "style": "jscs **/*.js --config=./.jscsrc.json",
     "quality": "plato -r -x 'node_modules|reports|test' -d reports/plato .",
-    "sass": "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./dist/css/app.css --include-path ./node_modules"
+    "sass": "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
+    "createPublicDir": "mkdir -p ./public/{js,css,images};",
```
Comment (JoeChapman) [inline package.json]: the only camel cased script name

**Comment by vaijab** [comment]
Cannot create directories:

`npm WARN cannot run in wd BRP@0.0.0 npm run createPublicDir; npm run sass; npm run browserify; npm run copy:images (wd=/brp_app)`

**Comment by JoeChapman** [comment]
:+1:

**Comment by JoeChapman** [comment]
btw, Ben made me add the thumbs up

**Comment by JoeChapman** [comment]
:eggplant:

**Comment by JoeChapman** [comment]
:octopus:

**Comment by easternbloc** [comment]
:shipit:

**Pull request merged**